supermarq-benchmarks/requirements.txt

```diff
@@ -1,4 +1,3 @@
 cirq-superstaq~=0.5.6
-matplotlib<=3.5.2
```

Comment:
I think we need this in supermarq-benchmarks.

Reply:
Used in https://github.com/Infleqtion/client-superstaq/blob/6f2ffe50658096b97062f5d9c3a5deff30938c77/supermarq-benchmarks/supermarq/plotting.py#L255

Reply:
it's already a requirement of cirq-core - all this was doing was limiting the max version (for reasons i can't really remember but which no longer seem relevant?)